build.gradle

```diff
@@ -227,9 +247,13 @@ integTest {
     // allows integration test classes to access test resource from project root path
     systemProperty('project.root', project.rootDir.absolutePath)
 
-    systemProperty "https", System.getProperty("https")
-    systemProperty "user", System.getProperty("user")
-    systemProperty "password", System.getProperty("password")
+    systemProperty("tests.opensearch.https", "true")
```

Comment:
I believe this should be `tests.opensearch.secure`

See security BWC tests here which spin up testclusters w/ security: https://github.com/opensearch-project/security/blob/main/.github/actions/run-bwc-suite/action.yaml#L47-L49